src/cli/run_command.zig

```diff
@@ -1611,6 +1611,7 @@ pub const RunCommand = struct {
         }
 
         if (log_errors) {
+            ctx.log.print(Output.errorWriter()) catch {};
```

Comment:
this is unrelated but wdyt should I keep it? otherwise any logged errors will never get seen right?

Reply:
It does seem correct to have this here, but it needs a test to ensure it works. I don't know a situation where it would print an error since this is the branch for failing to find the module. If you know one could you add a test? Otherwise let's delete it.